vivo-project/VIVO-languages
# DRAFT - Typo in add/edit forms for french version of "editor of"

Resolves : [https://jira.lyrasis.org/browse/VIVO-1797](https://jira.lyrasis.org/browse/VIVO-1797)

### What does this pull request do?

- Replaces concatenated variables to a single, syntactically agostic variable in addRoleToPersonTwoStage_fr_CA.ftl 
- Add variables assignation in addEditorRoleToPerson_fr_CA.ftl 
- Add variables assignation in following new files
    - addAttendeeRoleToPerson_fr_CAftl
   - addClinicalRoleToPerson_fr_CA.ftl
   - addEditorRoleToPerson_fr_CA.ftl
   - addHeadOfRoleToPerson_fr_CA.ftl
   - addMemberRoleToPerson_fr_CA.ftl
   - addOrganizerRoleToPerson_fr_CA.ftl
   - addOutreachProviderRoleToPerson_fr_CA.ftl
   - addResearcherRoleToPerson_fr_CA.ftl
   - addReviewerRoleToPerson_fr_CA.ftl
   - addServiceProviderRoleToPerson_fr_CA.ftl
   - addTeacherRoleToPerson_fr_CA.ftl
- Original files are located at [https://github.com/vivo-project/VIVO/tree/master/webapp/src/main/webapp/templates/freemarker/edit/forms](https://github.com/vivo-project/VIVO/tree/master/webapp/src/main/webapp/templates/freemarker/edit/forms)
- Lines removed typically read as 
<#assign GenericLabel_i18n = "${i18n().some_string}" />
- Lines added typically read as
<#assign roleAndGenericLabel_i18n = "${i18n().some_string}" />
<#assign nameAndGenericLabel_i18n = "${i18n().some_string}" />

- Add matching strings in vivo_all_fr_CA.properties

### How should this be tested?
- Add and Edit forms should be tested for the following sections:
   - Direction de
   - Membre de
   - Activités cliniques
   - Événements
   - Responsable de collection
   - Présentations
   - Autres activités de recherche
   - Activités d'enseignement
   - Évaluateur(-trice))
   - Organisateur d'événement
   - Activités professionnelles
   - Mobilisation des connaissances

### Interested parties

@MichelHeon
